content/docs/start/model-management/model-cicd.md

```diff
@@ -171,18 +242,40 @@ steps:
       echo "The right model is available and you can use the rest of this command to deploy it. Good job!"
 ```
 
+</tab>
+
+<tab title="GitLab">
+
+```yaml
+  script:
+
+  ...
+
+  # Install DVC
+  - pip install dvc
+  # Build commands to download and deploy the model
+  - dvc config --global studio.token ${DVC_STUDIO_TOKEN}
+  - dvc artifacts get  ${CI_REPOSITORY_URL} ${MODEL_NAME} --rev ${MODEL_VERSION}
```

Comment:
I think we need to better connect the "Downloading models" section at the top of this page with the CICD section down here. We could link between them, or mention how it's useful here. Other ideas?